libs/ui-lib-tests/cypress/integration/create-sno/0-create-cluster.cy.ts

```diff
@@ -46,7 +46,8 @@ describe(`Assisted Installer SNO Cluster Installation`, () => {
       commonActions.toNextStepAfter('Operators');
     });
 
-    it('Show the dev-preview badge for SNO', () => {
+    //We don't have OCP versions with dev-preview badge
```

Comment:
This is a consequence of removing version 4.8 from the fixtures, but we still have a  `SNODisclaimer` that could be shown if such a version existed.

So either:
1. Keep a test for showing the dev-preview badge and define a version for it.
2. Remove the test and the code for SNODisclaimer.